Retos/Reto #35 - PRIMEROS PASOS [Fácil]/python/KevinED11.py

```diff
@@ -0,0 +1,68 @@
+
+# Variables
+name: str = "kevin"
+age: int = 22
+money_in_the_bank: float = 1.0
+married: bool = True
+
+# Constant
+PI_VALUE = 3.1416
+
+# Conditionals
+if age >= 18:
+    print("eres mayor de edad")
+else:
+    print("eres menor de edad")
+
+if married:
+    print("esta casado")
+elif not married:
+    print("no esta casado")
+    
+# Structures
+names: list = ["kevin", "damian"]
+person: dict =  {"name": "kevin", "age": 22}
+unique_values: set = {"kevined11", "dani123", "kevined11"}
+person_values: tuple = (person.get("name"), person.get("age")) 
+
+
+# Loops
+for value in person_values:
+    print(value)
+
+counter = 1
+while counter <= 10:
+    print(counter)
+    counter += 1
+
+# functions
+def print_names(names: list) -> None:
+    for name in names:
+        print(names)
+
+def hello():
+    print("hello world")
+  
+def name1() -> str:
+    return "kevin"
+
+
+# Class
+class Person:
+    def __init__(self, name, age):
+        self.nam = name
+        self.age = age
+      
+# Exceptions
+def is_person(person: dict) -> dict:
```

Comment:
Hello, thank you for your contribution 👋

Seems to me that the function name is acting as a question, if so, can you return a boolean or reword the function name?

`is_person` the entity `person`?, [yes, no]
  - return: `bool`

As I said earlier, you could rename the function to something like this `check_person_entity`, `raise_if_not_person`. 

This comment is acting as a discussion, I am not trying to interfere in your contribution.